[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[DEBUG] JWT_ISSUER: https://aurorahours.com/identity-backend
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[DEBUG] JWT_ISSUER: https://aurorahours.com/identity-backend
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[INFO] /authorize started for client_id=browser-ui, state=7d694809-878d-4fbf-a0ed-0c47cf5c3ca1
[INFO] User 'u' authenticated, auth_code issued b27fa77a-d384-4fbd-a183-a4dcecf83ceb, state=7d694809-878d-4fbf-a0ed-0c47cf5c3ca1
[DEBUG] DEBUG: received client_secret: 'dev-client-secret'
[DEBUG] DEBUG: expected client_secret: 'dev-client-secret'
[INFO] Issued id_token for user=u client_id=browser-ui exp=2025-08-05 04:47:27.514521
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[INFO] /authorize started for client_id=browser-ui, state=8c5b6a67-409b-4529-ad8c-2e647f5478f1
Traceback (most recent call last):
  File "/home/auroraho/identity-backend/passenger_wsgi.py", line 1, in <module>
    from app import app as application
  File "/home/auroraho/identity-backend/app.py", line 6, in <module>
    from config import (
  File "/home/auroraho/identity-backend/config.py", line 16, in <module>
    FLASK_SECRET_KEY = get_required_env("FLASK_SECRET_KEY")
  File "/home/auroraho/identity-backend/config.py", line 13, in get_required_env
    raise Exception(f"{key} not set or still uses a placeholder!")
Exception: FLASK_SECRET_KEY not set or still uses a placeholder!
Traceback (most recent call last):
  File "/home/auroraho/identity-backend/passenger_wsgi.py", line 1, in <module>
    from app import app as application
  File "/home/auroraho/identity-backend/app.py", line 6, in <module>
    from config import (
  File "/home/auroraho/identity-backend/config.py", line 16, in <module>
    FLASK_SECRET_KEY = get_required_env("FLASK_SECRET_KEY")
  File "/home/auroraho/identity-backend/config.py", line 13, in get_required_env
    raise Exception(f"{key} not set or still uses a placeholder!")
Exception: FLASK_SECRET_KEY not set or still uses a placeholder!
[UID:2469][4031461] Child process with pid: 4031465 was killed by signal: 15, core dumped: no
Traceback (most recent call last):
  File "/home/auroraho/identity-backend/passenger_wsgi.py", line 1, in <module>
    from app import app as application
  File "/home/auroraho/identity-backend/app.py", line 6, in <module>
    from config import (
  File "/home/auroraho/identity-backend/config.py", line 16, in <module>
    FLASK_SECRET_KEY = get_required_env("FLASK_SECRET_KEY")
  File "/home/auroraho/identity-backend/config.py", line 13, in get_required_env
    raise Exception(f"{key} not set or still uses a placeholder!")
Exception: FLASK_SECRET_KEY not set or still uses a placeholder!
Traceback (most recent call last):
  File "/home/auroraho/identity-backend/passenger_wsgi.py", line 1, in <module>
    from app import app as application
  File "/home/auroraho/identity-backend/app.py", line 6, in <module>
    from config import (
  File "/home/auroraho/identity-backend/config.py", line 16, in <module>
    FLASK_SECRET_KEY = get_required_env("FLASK_SECRET_KEY")
  File "/home/auroraho/identity-backend/config.py", line 13, in get_required_env
    raise Exception(f"{key} not set or still uses a placeholder!")
Exception: FLASK_SECRET_KEY not set or still uses a placeholder!
[UID:2469][4031600] Child process with pid: 4031609 was killed by signal: 15, core dumped: no
Traceback (most recent call last):
  File "/home/auroraho/identity-backend/passenger_wsgi.py", line 1, in <module>
    from app import app as application
  File "/home/auroraho/identity-backend/app.py", line 6, in <module>
    from config import (
  File "/home/auroraho/identity-backend/config.py", line 16, in <module>
    FLASK_SECRET_KEY = get_required_env("FLASK_SECRET_KEY")
  File "/home/auroraho/identity-backend/config.py", line 13, in get_required_env
    raise Exception(f"{key} not set or still uses a placeholder!")
Exception: FLASK_SECRET_KEY not set or still uses a placeholder!
[UID:2469][4031600] Child process with pid: 4031653 was killed by signal: 15, core dumped: no
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[INFO] /authorize started for client_id=browser-ui, state=10825be3-4927-49ee-becb-b086c74a27d1
[INFO] User 'u' authenticated, auth_code issued 0ad3a818-8dcc-4acb-9246-d0b4e7115c84, state=10825be3-4927-49ee-becb-b086c74a27d1
[DEBUG] DEBUG: received client_secret: 'dev-client-secret'
[DEBUG] DEBUG: expected client_secret: 'dev-client-secret'
[INFO] Issued id_token for user=u client_id=browser-ui exp=2025-08-05 05:01:54.720515
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[DEBUG] JWT_ISSUER: https://aurorahours.com/identity-backend
[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[INFO] /authorize started for client_id=browser-ui, state=fea2506f-857c-45be-bbad-220fda1b971b
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[INFO] /Using auth code DB at:/home/auroraho/identity-backend/authcodes.db
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[INFO] User 'u' authenticated, auth_code issued c531f557-79b9-424b-b562-d49384fe6dfb, state=fea2506f-857c-45be-bbad-220fda1b971b
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[DEBUG] DEBUG: received client_secret: 'dev-client-secret'
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[DEBUG] DEBUG: expected client_secret: 'dev-client-secret'
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[INFO] Issued id_token for user=u client_id=browser-ui exp=2025-08-05 05:24:26.902881
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

[INFO] /authorize started for client_id=browser-ui, state=cba6db01-f0e7-49fb-a0e0-37157632f0f2
[WARNING] Central logging failed: HTTP 404 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

